### DISCUSSION TITLE
Negativer Verbrauch (Balkonkraftwerk)

### DISCUSSION THREAD

**Discussion by pkuehnel** [camueller/SmartApplianceEnabler#606] [Q&A] [answered]
Hat schon mal jemand probiert negative Verbrauchs Werte zu schicken? Ich hab aktuell das Problem, dass ich ein bestehendes Balkonkraftwerk irgendwie ins Sunnyportal reinbringen möchte, aber keine Ahnung wie und das ist noch das einzig irgendwie sinnvolle.

**Comment by Trunks1982** [comment]
Soll dir das als Verbraucher angezeigt werden?
Anders gehts nicht.

**Reply by pkuehnel** [reply]
Dass es als Verbraucher angezeigt wird, ist mir bewusst, meine Frage ist, ob das mit negativen Werten funktioniert.

**Reply by Trunks1982** [reply, marked as answer]
Nein
